docs/src/main/sphinx/develop/supporting-merge.rst

```diff
@@ -344,9 +344,9 @@ methods.
         ConnectorTableHandle tableHandle)
 
   This method is called as the engine starts processing a ``MERGE`` statement.
-  The connector must return a ``RowChangeParadigm`` enum instance.  If the
-  connector does not support ``MERGE`` it should throw a ``NOT_SUPPORTED``
-  exception, meaning that SQL ``MERGE`` is not supported by the connector.
+  The connector must return a ``RowChangeParadigm`` enumeration instance. If the
```

Comment:
Let's improve the second sentence:

> If the connector doesn't support `MERGE`, then it should throw a `NOT_SUPPORTED` exception to indicate that SQL `MERGE` is not supported by the connector. Note that the default implementation already throws this exception when the method is not implemented.

Reply:
Done. Note that the Google style guide allows (prefers?) `isn't` instead of `is not`: https://developers.google.com/style/contractions